apps/hasura-express/.env

```diff
@@ -0,0 +1,17 @@
+# App
+
+PORT=3000
+APP_URL=http://localhost:3000
+
+# BoxyHQ SAML
+
+REDIRECT_URL=http://localhost:3000/
+SAML_AUDIENCE=https://saml.boxyhq.com
+DB_ENGINE=sql
+DB_URL=
+DB_TYPE=
+
+# HASURA
+
+HASURA_JWT_SECRET=497254393080ecba7b4b4208fc4e0bbe000c6bfe58e2a83a50a630d0d8700833
```

Comment:
let's align with the hasura-next-js example

Reply:
Done